Filter/StringFilter.php

```diff
@@ -25,28 +25,39 @@ public function filter(ProxyQueryInterface $proxyQuery, $alias, $field, $data)
             return;
         }
 
-        $data['value'] = trim($data['value']);
+        $value = $data['value'] = trim($data['value']);
```

Comment:
ok, valid point. but then i'd just keep using $data['value'] below for simplicity. having a local variable does not add much